app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -0,0 +1,223 @@
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * Message Card.
+ *
+ * @param message [Message] that holds a representation of GleanPlum message from Nimbus.
+ * @param onClick Invoked when user clicks on the message card.
+ * @param onCloseButtonClick Invoked when user clicks on close button to remove message.
+ */
+@Suppress("LongMethod")
+@Composable
+fun MessageCard(
+    message: Message,
+    onClick: () -> Unit,
+    onCloseButtonClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .padding(
+                top = 16.dp,
+                bottom = 16.dp
+            )
+            .then(
+                if (message.data.buttonLabel.isNullOrBlank()) {
+                    Modifier.clickable(onClick = onClick)
+                } else {
+                    Modifier
+                }
+            ),
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layer2,
+    ) {
+        Column(
+            Modifier
+                .padding(all = 16.dp)
+                .fillMaxWidth()
+        ) {
+            val title = message.data.title
+            if (!title.isNullOrBlank()) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    SectionHeader(
+                        text = title,
+                        modifier = Modifier
+                            .weight(1f)
+                    )
+
+                    Spacer(modifier = Modifier.height(10.dp))
+
+                    IconButton(
+                        modifier = Modifier.size(20.dp),
+                        onClick = onCloseButtonClick
+                    ) {
+                        Icon(
+                            painter = painterResource(R.drawable.ic_close),
+                            contentDescription = stringResource(
+                                R.string.remove_home_collection_placeholder_content_description
+                            ),
+                            tint = FirefoxTheme.colors.iconPrimary
+                        )
+                    }
+                }
+
+                PrimaryText(
+                    text = message.data.text,
+                    modifier = Modifier.fillMaxWidth(),
+                    fontSize = 14.sp
+                )
+            } else {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    PrimaryText(
+                        text = message.data.text,
+                        modifier = Modifier.weight(1f),
+                        fontSize = 14.sp
+                    )
+
+                    IconButton(
+                        modifier = Modifier.size(20.dp),
+                        onClick = onCloseButtonClick
+                    ) {
+                        Icon(
+                            painter = painterResource(R.drawable.ic_close),
```

Comment:
```suggestion
                            painter = painterResource(R.drawable.mozac_ic_close_20),
```

Can you also change this over in `CollectionsPlaceholder`?